examples/minimal-static-router/src/classifiers.rs

```diff
@@ -121,31 +121,32 @@ mod tests {
     use route_rs_packets::EthernetFrame;
     use route_rs_runtime::link::primitive::ClassifyLink;
     use route_rs_runtime::link::LinkBuilder;
-    use route_rs_runtime::utils::test::harness::{run_link, initialize_runtime};
+    use route_rs_runtime::utils::test::harness::{initialize_runtime, run_link};
     use route_rs_runtime::utils::test::packet_generators::immediate_stream;
 
     #[test]
     fn classify_ip() {
-        let runtime = initialize_runtime();
-        runtime.spawn(async {
-            let data_v4: Vec<u8> = vec![
-                0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 0x08, 00, 0x45, 0, 0, 20, 0, 0,
-                0, 0, 64, 17, 0, 0, 192, 178, 128, 0, 10, 0, 0, 1,
-            ];
-            let data_v6: Vec<u8> = vec![
-                0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 0x86, 0xDD, 0x60, 0, 0, 0, 0, 4,
-                17, 64, 0xde, 0xad, 0xbe, 0xef, 0xde, 0xad, 0xbe, 0xef, 0xde, 0xad, 0xbe, 0xef, 0xde,
-                0xad, 0xbe, 0xef, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 0xa, 0xb, 0xc,
-                0xd,
-            ];
-            let data_unknown: Vec<u8> = vec![
-                0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 0xff, 0xff,
-            ];
-            let frame1 = EthernetFrame::from_buffer(data_v4, 0).unwrap();
-            let frame2 = EthernetFrame::from_buffer(data_v6, 0).unwrap();
-            let frame3 = EthernetFrame::from_buffer(data_unknown, 0).unwrap();
+        let data_v4: Vec<u8> = vec![
+            0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 0x08, 00, 0x45, 0, 0, 20, 0, 0,
+            0, 0, 64, 17, 0, 0, 192, 178, 128, 0, 10, 0, 0, 1,
+        ];
+        let data_v6: Vec<u8> = vec![
+            0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 0x86, 0xDD, 0x60, 0, 0, 0, 0, 4,
+            17, 64, 0xde, 0xad, 0xbe, 0xef, 0xde, 0xad, 0xbe, 0xef, 0xde, 0xad, 0xbe, 0xef, 0xde,
+            0xad, 0xbe, 0xef, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 0xa, 0xb, 0xc,
+            0xd,
+        ];
+        let data_unknown: Vec<u8> = vec![
+            0xde, 0xad, 0xbe, 0xef, 0xff, 0xff, 1, 2, 3, 4, 5, 6, 0xff, 0xff,
+        ];
+        let frame1 = EthernetFrame::from_buffer(data_v4, 0).unwrap();
+        let frame2 = EthernetFrame::from_buffer(data_v6, 0).unwrap();
+        let frame3 = EthernetFrame::from_buffer(data_unknown, 0).unwrap();
+
+        let mut runtime = initialize_runtime();
+        let results = runtime.block_on(async {
```

Comment:
We may want to encapsulate this pattern into the test harness later

Reply:
Agree